plone/rfc822/__init__.py

```diff
@@ -1,4 +1,13 @@
 # -*- coding: utf-8 -*-
+import sys
+PY3 = sys.version_info[0] == 3
+if PY3:
+    text_type = str
+    binary_type = bytes
+else:
+    text_type = unicode
+    binary_type = str
```

Comment:
@pbauer isn't [six](https://pypi.python.org/pypi/six) meant for this kind of things?